ext/pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[build-system]
+build-backend = "hatchling.build"
+requires = ["hatchling"]
+
+[project]
+name = "django-stubs-ext"
+# NB! For clarity, keep version major.minor.patch in sync with django-stubs.
+# It's fine to skip django-stubs-ext releases, but when doing a release, update this to newest django-stubs version.
+version = "5.2.0"
+description = "Monkey-patching and extensions for django-stubs"
+readme = "README.md"
+license = "MIT"
+license-files = ["LICENSE.md"]
+maintainers = [
+  { name = "Marti Raudsepp", email = "marti@juffo.org" },
+]
+authors = [
+  { name = "Simula Proxy" },
+]
+requires-python = ">=3.10"
+classifiers = [
+    "License :: OSI Approved :: MIT License",
```

Comment:
I believe that the license trove classifiers are deprecated in favour of an SPDX license expression in `license`, so this could be removed:

```suggestion
```

Reply:
ye I think this is true, I've seen adam do that [in djade ](https://github.com/adamchainz/djade/pull/123)

I've removed it